Initium-Odp/war/WEB-INF/odppages/ajax_sublocations.jsp

```diff
@@ -1,21 +1,23 @@
 <%@ taglib uri='http://java.sun.com/jsp/jstl/core' prefix='c'%>
 <%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1"%>
 
-<c:if test='${showHidden==false}'>
-	<p><a onclick='viewLocalNavigation(event, true)'>Show Hidden Paths</a></p>
-</c:if>
-<c:if test='${showHidden==true}'>
-	<p><a onclick='viewLocalNavigation(event, false)'>Hide Hidden Paths</a></p>
-</c:if>
-<c:forEach var="location" items="${locations}">
-	<div onclick='doGoto(event, ${location.pathId})' class='standard-button-highlight sublocation-display-container location-link-${location.locationId}'>
-		<div class='sublocation-display' style='background-image:url(${location.bannerUrl})'></div>
-		<h5>${location.name}</h5>
-		<c:if test="${location.isForgettable=='true'}">
-			<div class='standard-button-highlight forget-button' onclick='doForgetCombatSite(event,${location.locationId})'>X</div>
-		</c:if>
-	</div>
-</c:forEach>
-<c:if test='${forgetAllCombatSitesHtml!=null }'>
-${forgetAllCombatSitesHtml}
+<c:if test='${locationsViewable==true }'>
```

Comment:
I'm only seeing this attribute set to false when character is incapacitated, meaning it's never set to true. Make sure this is set to true in the default case of SublocationsController.